pkg/api/api.go

```diff
@@ -104,6 +104,7 @@ var (
 	errFileStore            = errors.New("could not store file")
 	errInvalidPostageBatch  = errors.New("invalid postage batch id")
 	errBatchUnusable        = errors.New("batch not usable")
+	errNotImplemented       = errors.New("not implemented")
```

Comment:
This error is not the same type as you've defined in the `devnode.go` file. You should drop this error, export the `errNotImplemented` from the `devnode.go` file, and check against this exported error.